app/controllers/concerns/foreman/controller/parameters/trend.rb

```diff
@@ -3,7 +3,7 @@ module Foreman::Controller::Parameters::Trend
 
   class_methods do
     def trend_params_filter
-      Foreman::ParameterFilter.new(::Trend).tap do |filter|
+      Foreman::ParameterFilter.new(ForemanStatistics::Trend).tap do |filter|
```

Comment:
I believe `ForemanStatistics::Trend` might be undefined when this module is loaded without the plugin installed

Reply:
only if the method is used and that should not happen if the plugin is not around due to `ensure_plugin` guard on the controller